shopfloor/services/checkout.py

```diff
@@ -821,74 +893,101 @@ def scan_package_action(self, picking_id, selected_line_ids, barcode):
         search = self._actions_for("search")
 
         selected_lines = self.env["stock.move.line"].browse(selected_line_ids).exists()
+        search_result = search.find(
+            barcode,
+            types=(
+                "package",
+                "product",
+                "packaging",
+                "lot",
+                "serial",
+                "delivery_packaging",
```

Comment:
If we want to preserve the previous implementation the package was handled later, but you probably put it first for some reasons. It could bring regressions only if the package names collide with other types (low chance). Ignore my comment if you think it's OK.
```suggestion
                "product",
                "packaging",
                "lot",
                "serial",
                "package",
                "delivery_packaging",
```

Reply:
package is usually more precise than product...